### PR TITLE
Change centrifuge build to export a directory of all the files (like …

### PR DESCRIPTION
**Pull request by jfy133**
…other reference building/database building modules)

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/centrifuge/build/main.nf [hunk 1]
```diff
@@ -15,21 +15,23 @@ process CENTRIFUGE_BUILD {
...
+    tuple val(meta), path("${prefix}/") , emit: cf
     path "versions.yml"           , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
+    prefix = task.ext.prefix ?: "${meta.id}"
```
Comment (sofstam) [inline modules/nf-core/centrifuge/build/main.nf:26]: No def is needed here?

File: modules/nf-core/centrifuge/build/main.nf [hunk 2]
```diff
@@ -15,21 +15,23 @@ process CENTRIFUGE_BUILD {
...
     def size_table_cmd = size_table ? "--size_table ${size_table}" : ""
     """
+    mkdir ${prefix}
+
     centrifuge-build \\
         -p $task.cpus \\
         $fasta \\
-        ${prefix} \\
+        ${prefix}/${prefix} \\
```
Comment (sofstam) [inline modules/nf-core/centrifuge/build/main.nf:34]: A bit confused why this should be written that way.

**Review by sofstam — APPROVED** [review]

File: modules/nf-core/centrifuge/build/main.nf (hunk 1 above)
Comment (jfy133) [reply modules/nf-core/centrifuge/build/main.nf:26]: No, otherwise can't access 'prefix' i nteh output scope above :)

**Comment by jfy133** [comment]
I will force merge because singularity downloads running extremely slowly as the other tests are passing

**Pull request merged**